content/en/tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-state.md

```diff
@@ -26,52 +26,28 @@ Before you start building states, read [Key concepts: States](../../../../../pla
 
 The easiest way to write any CorDapp is to start from a template. This ensures that you have the correct files to begin building.
 
-1. Navigate to the Kotlin and Java template repositories and decide which you'd like to clone:
-  * https://github.com/corda/cordapp-template-kotlin
-  * https://github.com/corda/cordapp-template-java
+1. Clone the [Java template repository](https://github.com/corda/cordapp-template-java).
+
 
 2. Open a terminal window in the directory where you want to download the CorDapp template.
 
-3. Run the following command:
 
-   {{< tabs name="tabs-1" >}}
-   {{% tab name="kotlin" %}}
-   ```kotlin
-   git clone https://github.com/corda/cordapp-template-kotlin.git
-   ```
-   {{% /tab %}}
+3. Run the following command:
```

Comment:
> Instructions for clone either the kotlin or java version but then the tutorial code is only in Java
From [the ticket](https://r3-cev.atlassian.net/browse/DOC-3867) -> I've chosen to take out the kotlin to reflect this

Reply:
Please do not remove link to the Kotlin repo. It's essential that users have access to the Kotlin code. Instead, add a not saying that the following tutorial is for Java only.